src/Store/VimStoreConnector.re

```diff
@@ -317,17 +319,59 @@ let start =
       open Vim.BufferUpdate;
       Log.info("Vim - Buffer update: " ++ string_of_int(update.id));
       open Core.Types;
+      open Model.State;
+
+      let isFull = update.endLine == (-1);
+
+      let bufferMaybe =
```

Comment:
The convention is usually `maybeBuffer` I think. At least from what I've seen. It also reads better to me.

Good descriptive naming either way though :+1: